src/app/testing/map-testing/share-link-item-test.utils.ts

```diff
@@ -1,9 +1,8 @@
 import {ShareLinkItem} from '../../shared/interfaces/share-link.interface';
 import {MinimalGeometriesUtils} from './minimal-geometries.utils';
-import {TimeExtentUtils} from '../../shared/utils/time-extent.utils';
 
 export class ShareLinkItemTestUtils {
-  public static createShareLinkItem(): ShareLinkItem {
+  public static createShareLinkItem(timeExtentStart: Date, timeExtentEnd: Date): ShareLinkItem {
```

Comment:
First of all: no utils no worries.
Secondly: It seems weird that the `createSharelinkItem` method has the time extent start/end date as arguments. I know that it's probably a side effect from using the time service instead. However, wouldn't it be a little bit cleaner to hand over this service instead of those values?

Reply:
Okay, forget my comment :D I just saw that this is a test helper class.